web.Dockerfile

```diff
@@ -1,12 +1,11 @@
 FROM node:carbon
 
 COPY .env .env
-COPY web/package.json web/package.json
-COPY web/yarn.lock web/yarn.lock
+COPY docs docs
 
-WORKDIR web
-RUN yarn install --production --silent
-COPY web .
+RUN yarn global add http-server
+
+RUN echo "source ./.env ; export NODE_ENV=development ; http-server ./docs -a 0.0.0.0 -p 8080 -c-1" > entry.sh
```

Comment:
I like how you piped this, saves us from having another file in the main directory 💯